json/api/request.go

```diff
@@ -0,0 +1,58 @@
+/*
+* Copyright(c) 2012-2021 Intel Corporation
+* SPDX-License-Identifier: BSD-3-Clause-Clear
+ */
+
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+)
+
+type Request struct {
+	Request_list_caches bool               `json:"request list_caches"`
+	Request_get_stats   bool               `json:"request statistics"`
+	Get_stats_info           Stats_request_info `json:"statistics request info"`
+}
```

Comment:
I think it would be better to have common request header (it may consist of a single field like "command", which would contain name of the command) and the rest of the request structure would be different for each command. Otherwise we will end up with request structure containing all the input for all variants of all commands.

That would require preparsing request json as a `Header` struct which may look like this:
```go
type Header struct {
    Command string `json:"command"`
}
```
That would allow us to determine the command type, and then we would be able to parse the request as a specific command.